docs/preview/features/secrets/consume-from-key-vault.md

```diff
@@ -4,15 +4,25 @@ layout: default
 ---
 
 # Consuming Azure Key Vault secrets
```

Comment:
```suggestion
# Working with Azure Key Vault 
```

I'm now missing a bit documentation on 'consuming' Key Vault secrets (in my opinion, consuming is retrieving secrets).
So, if we write a section here on storing secrets, I 'd make the title cover that by changing it from 'consuming' to 'working with'.

Next to that, I would also add a section here on retrieving secrets ?  I looked briefly at the documentation, and the section on 'retrieving' secrets is now somewhat mingled with a section on how to develop your own secret provider

Reply:
> I'm now missing a bit documentation on 'consuming' Key Vault secrets (in my opinion, consuming is retrieving secrets).
So, if we write a section here on storing secrets, I 'd make the title cover that by changing it from 'consuming' to 'working with'.

Yes, we can definitely change that here, then. The file name would stays the same, though. Maybe we should make it more clear that this is something 'extra' otherwise by saying 'working with ...' is a bit unclear what the diff would be with the section on the actual Azure Key Vault secret provider.

> Next to that, I would also add a section here on retrieving secrets ? I looked briefly at the documentation, and the section on 'retrieving' secrets is now somewhat mingled with a section on how to develop your own secret provider

I think the section you're looking for here is this: https://security.arcus-azure.net/features/secrets/general , where we talk about consuming secrets.

Reply:
Ok.  It is a bit abstract for me now to conclude.
I can go over the docs once it is build I guess.

Reply:
Ok, we can take a look then.